lambdaconf-2015/index.html

```diff
@@ -1231,6 +1231,23 @@ <h1 id="schedule-h1">Schedule</h1>
                   <td></td>
                 </tr>
               </tr>
+
+              <tr>
+                <td>None</td>
+                <td class="title">The Case for the Cutting-Edge</td>
+                <td><a href="#matt-farmer">Matt Farmer</a></td>
+              </tr>
+              <tr>
+                <td>25m</td>
```

Comment:
We call these 30m from the perspective of attendees. Other than that, looks great!